packages/astro/src/content/types-generator.ts

```diff
@@ -5,6 +5,7 @@ import glob from 'fast-glob';
 import { bold, cyan } from 'kleur/colors';
 import { type ViteDevServer, normalizePath } from 'vite';
 import { z } from 'zod';
+import { zodToTs, printNode } from 'zod-to-ts';
```

Comment:
We'll have to document the limitations of this library, iirc some types can't be generated